Assets/HoloToolkit/Utilities/Scripts/FileSystemHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if UNITY_EDITOR
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    public static class FileSystemHelper
```

Comment:
Please add class descriptions for your new classes and update the README in the Utilities folder.